scripts/mixins/families/antlion_ambush.lua

```diff
@@ -15,12 +15,21 @@ g_mixins.families.antlion_ambush = function(antlionAmbushMob)
     antlionAmbushMob:addListener("ENGAGE", "ANTLION_AMBUSH_ENGAGE", function(mob, target)
         mob:useMobAbility(278) -- Pit Ambush
         mob:setMobMod(xi.mobMod.NO_MOVE, 0)
+        mob:timer(2000, function(ant)
+            ant:hideName(false)
+            ant:setUntargetable(false)
+            ant:setAnimationSub(1)
+        end)
     end)
     antlionAmbushMob:addListener("DISENGAGE", "ANTLION_AMBUSH_DISENGAGE", function(mob)
         mob:hideName(true)
         mob:setUntargetable(true)
         mob:setAnimationSub(0)
         mob:wait(2000)
+        mob:timer(2000, function(ant)
+            local spawn = mob:getSpawnPos()
+            mob:setPos(spawn.x, spawn.y, spawn.z, spawn.rot)
```

Comment:
This doesn't seem to happen on retail.

It appears that when they go back underground, they stay right there for awhile. if they were to far from "home" (pop area) and are non NM they can despawn instead (sometimes before they go underground). 

NMs (or at least the one I checked) appear to "walk home" _while underground_ o.O which seams like such a pain in the butt.